compliance/model/src/test/java/org/eclipse/rdf4j/model/util/IsomorphicBenchmark.java

```diff
@@ -59,6 +64,7 @@
 	private Model empty_2 = getModel("empty.ttl");
 	private Model blankNodes_2 = getModel("blankNodes.ttl");
 	private Model shacl_2 = getModel("shacl.ttl");
+	private Model shaclValidationReport_2 = getModel("shaclValidationReport.ttl");
```

Comment:
The sorting I was doing was what was causing the isomorphic check on this file to take so long. Thought 6 seconds was kinda slow, it was nothing in comparison to how slow it was when I sorted on object.

Reply:
I'm aware, which is why I added the sorted version as a unit test in IsomorphicTest. It has a timeout of 2 seconds on it.